tests/demo-todo-app.spec.ts

```diff
@@ -1,24 +1,29 @@
 import { test, expect, type Page } from "@playwright/test";
-
-test.describe.configure({ mode: "serial" });
-
-test.beforeEach(async ({ page }) => {
-  await page.goto("/");
+import CouchDB from "../src/lib/CouchDB";
+const db = new CouchDB({
+  user: "admin",
+  pass: "admin",
+  dbName: "test",
+  logLevelEnv: "CRITICAL",
 });
 
-test.afterEach(async ({ page }) => {
-  const authHeader = {
-    Authorization: `Basic ${btoa("admin:admin")}`,
-  };
-  const resp = await fetch("http://localhost:5984/test", {
-    method: "DELETE",
-    headers: new Headers(authHeader),
-  });
-  const resp2 = await fetch("http://localhost:5984/test", {
-    method: "PUT",
-    headers: new Headers(authHeader),
-  });
-});
+test.beforeEach(
+  async ({ page, context, browserName }, { line, column, workerIndex }) => {
+    const dbName = `${browserName}-${line}-${column}-${workerIndex}`;
+    await db.createDatabase({ dbName });
+    await context.setExtraHTTPHeaders({
+      db_name: dbName,
+    });
+    await page.goto("/");
+  }
+);
+
+test.afterEach(
+  async ({ page, context, browserName }, { line, column, workerIndex }) => {
+    const dbName = `${browserName}-${line}-${column}-${workerIndex}`;
+    db.deleteDatabase({ dbName });
+  }
+);
```

Comment:
Using the `browserName`, `line`, `column`, and `workerIndex` to create unique databases for each test and then deleting them after each test.

Also setting the extra header so that the api endpoints know which db to work with.